tests/llama_index/test_llama_index_tracer.py

```diff
@@ -122,6 +122,19 @@ def test_trace_llm_complete_stream():
     assert attr["model_dict"]["model"] == model_name
 
 
+def _get_llm_input_content_json(content):
+    if Version(llama_index.core.__version__) >= Version("0.12.5"):
```

Comment:
nit: Can you add a comment for what was changed and why this condition is required?